projects/ujson/build.sh

```diff
@@ -18,6 +18,9 @@
 # Build and install project (using current CFLAGS, CXXFLAGS).
 pip3 install .
 
+# Install Hypothesis for the structured fuzz harness
+pip3 install hypothesis
```

Comment:
Should this be done in the base-builder image ?

Reply:
Maybe! It's a fairly lightweight package (only two transitive dependencies), but that's still a bunch of code which most fuzzers don't need in the container... or the convenience might win out.

I think most Hypothesis-based fuzz harnesses will have other dependencies though, such as Hypothesmith, ao it makes sense to install it explicitly here.

Reply:
In this case, maybe it is better in Dockerfile than build.sh, what do you think ?

Reply:
Yes please move this to Dockerfile. If many projects start using this, then we can move this install to base-builder (similar to Atheris).